src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const STATE_HISTORY = [];
+  const CLONED_STATE = { ...state };
+
+  for (const ACTION of actions) {
```

Comment:
The variable `ACTION` should follow camelCase naming convention. Consider renaming it to `action`.